### PR TITLE
Add comment to course reset

### PR DESCRIPTION
**Pull request by hajorg**
### Summary
This PR allows for comment to be added when resetting a learner's course
See: https://2u-internal.atlassian.net/browse/AU-1935

Screenshot 2024-03-27 at 14 27 03
Screenshot 2024-03-27 at 14 28 41

### CODE REVIEW

**Review by jansenk — APPROVED** [review]
LGTM

File: src/users/CourseReset.jsx [hunk 1]
```diff
@@ -17,8 +17,20 @@ function CourseReset({ username, intl }) {
   const [courseResetData, setCourseResetData] = useState([]);
   const [error, setError] = useState('');
   const [isOpen, open, close] = useToggle(false);
+  const [comment, setComment] = useState();
```
Comment (AfaqShuaib09) [inline src/users/CourseReset.jsx]:
```suggestion
const [comment, setComment] = useState('');
```

File: src/users/CourseReset.jsx [hunk 2]
```diff
@@ -17,8 +17,20 @@ function CourseReset({ username, intl }) {
...
+  const [commentError, setCommentError] = useState('');
   const POLLING_INTERVAL = 10000;
 
+  const handleCommentChange = (e) => {
+    const text = e.target.value;
+    setComment(text);
+    if (text.length > 255) {
```
Comment (AfaqShuaib09) [inline src/users/CourseReset.jsx]: would be nice if we store this 255 in a variable
```javascript
MAX_COMMENT_LENGTH = 255
```
so we can use this in comment message error too

**Review by AfaqShuaib09 — APPROVED** [review]
Just a couple of suggestions otherwise looks good to me 👍🏻

**Pull request merged**